airbyte-webapp/src/components/ui/Text/Text.tsx

```diff
@@ -14,14 +14,16 @@ interface TextProps {
   bold?: boolean;
   inverseColor?: boolean;
   title?: string;
+  gradient?: boolean;
```

Comment:
Added `gradient` as a prop to make the blue-to-orange text color gradient reusable